swe_cp.m

```diff
@@ -17,7 +17,8 @@ function swe_cp(SwE)
 % the following analagous maps:
 %
 %   - swe_vox_mask: The mask image for the analysis.
-%   - swe_vox_beta_c: The beta map for each contrast
+%   - swe_vox_beta_b: The beta map.
+%   - swe_vox_beta_c: The beta map for each contras.
```

Comment:
Let's call these 'contrasts' or 'con'... i.e. `swe_vox_con_c{c#}` (they're not beta's... they're linear combinations of betas)

Reply:
Honestly... I cannot see how `ess` exists for the F stat code (just poured over `swe_contrasts.m`... don't worry about it for F stats... just `con` for T stats.

Reply:
I have renamed the `beta_c{c#}` files but am unsure what you mean for me to do with the F statistic maps... what do you mean by `ess`?

Reply:
Don't worry about it.  For reference:  A F-test can be defined by
> F = (SSreg-SSreg0)/(SStot-SSreg) / ( (P-P0)/(N-P) ) = ESS/Err / (df1/df2)
i.e. the "extra sum of squares" explained by the full model relative to a reduced(null) model.  SPM writes out this "extra sum of squares", i.e. (SSreg-SSreg0) = ESS by analogy to the `con` images being the numerator of a T statistic.

I poured over the SWE code but I cannot find for my life an exact form for ESS... so we'll leave it for Bryan :)

Reply:
Ah right yes! I see now! Yes... I'm not sure on the exact form there either.